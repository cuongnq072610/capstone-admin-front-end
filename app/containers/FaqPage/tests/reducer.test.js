import { fromJS } from 'immutable';
import faqPageReducer from '../reducer';

describe('faqPageReducer', () => {
  it('returns the initial state', () => {
    expect(faqPageReducer(undefined, {})).toEqual(fromJS({}));
  });
});
