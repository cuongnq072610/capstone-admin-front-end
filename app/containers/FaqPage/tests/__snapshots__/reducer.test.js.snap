// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`faqPageReducer returns the initial state 1`] = `Immutable.Map {}`;
